cmd/run/socket/proxy.go

```diff
@@ -230,24 +241,98 @@ func (p *proxy) proxyHTTP(cli, srv *bufConn) error {
 	protocol := guessProtocol(sample)
 	switch protocol {
 	case "http/1":
+		return p.proxyHTTP1(cli, srv)
 	case "http/2":
+		// TODO: reintroduce HTTP/2 support
+		return p.proxyFallback(cli, srv)
 	default:
 		return p.proxyFallback(cli, srv)
```

Comment:
Were we just always falling back earlier? Not sure I understand the semantics here

Reply:
No, I removed HTTP/2 support temporarily.